kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
+  - ambassador/base
```

Comment:
Shouldn't there be a kustomization file for each package so that each application can be installed independently?

Reply:
Yes, there is one for each component.
This is to combine everything and deploy the set of components. 
I've explained the reasoning here: https://github.com/kubeflow/manifests/issues/4

Reply:
For each platform we will have different packages that we want to install; e.g. there are GCP specific packages. 
We also have different configurations for each platform;
e.g.
GCP + IAP
GCP + Basic Auth
Minikube

For each of the above there is a different set of applications to install and possibly different overlays to be used for each application.

Is the intent for the root kustomization.yaml to support all of that? Or is the root kustomization.yaml only going to provide a vanilla K8s cconfig.

Reply:
Yeah, the root customization can choose what components to install from the different components. 
So for minikube/gke, we'd have an overlay based on the root customization.

Reply:
Cool.